scipy/stats/_continuous_distns.py

```diff
@@ -3582,56 +3583,943 @@ class levy_stable_gen(rv_continuous):
+    for pdf calculation. Setting the threshold to ``None`` (default) will disable FFT. 
+    For cdf calculations the Zolatarev method is superior in accuracy, so FFT is disabled by default.
+    
+    The supported methods for fitting the estimates of the parameters under parameterization A are  
+    "quantile" method (see [MC]) and the estimation based on using "empirical characteristic function", (see [KO]). 
+    MLE estimation of parameters in fit method uses the quantile method initially. 
+    Note that MLE doesn't always converge if using FFT for pdf calculations; 
+    so it's best that ``pdf_fft_min_points_threshold`` is left unset or set to default value of 100. 
+
+    %(after_notes)s
+    This implementation supports also 3 more parameterizations, which 
+    arise by representing complex number z in polar coordinates "r ..math::\exp{ i ..math::\theta }". 
+    The characteristic functions take the following form: 
+    - Parameterization B: 
+
+    .. math::         
+        \varphi(t; \alpha, \beta_B, c_B, \mu_B) = e^{c_B(it\mu_B -|k|^{\alpha }\Phi_B(\alpha, k))}
+
+        where:
+
+    .. math::
+        \Phi_B(t, \alpha) = \begin{cases}
+                    e^{-i\beta_B K(\alpha) \operatorname{sign}(t)\pi/2} &\alpha \neq 1\\
+                    \frac {2}{\pi}+i\beta_B\log|t|\operatorname{sign}(t) & \alpha = 1
+                    \end{cases}
+    and:
+              
+    .. math::
+        K(\alpha) = \alpha -1 + sgn(1 - \alpha), 
+        -1 < \beta_B K(\alpha) < 1, 
+        and the transformation relationship between parameterization A and B are given by: 
+
+    ..math::
+        \beta_B K(\alpha) = \frac{2}{\pi} atan(\beta_A \tan[\pi\alpha / 2])
+        c_B = \frac{c_A}{\cos[\beta_B K(\alpha)\pi/2]}
+        mu_B = mu_A \cos[\beta_B K(\alpha)\pi/2]
+
+    - Parameterization C:
+    .. math::         
+        \varphi(t; \alpha, \delta, c_C, \mu_C) = e^{ c_C(it\mu_C -|t|^{\alpha}e^{-i\alpha \delta \operatorname{sign}(t)\pi/2}}
+
+    where:
+              
+    ..math::
+         \delta =\frac{\beta_B K(alpha)}{\alpha} = \frac{2}{\alpha \pi} atan(\beta_A \tan[\pi\alpha / 2])
+         c_C = c_B = \frac{c_A}{\cos[\beta_B K(\alpha)\pi /2}
+ 
+    - Parameterization P:
+    .. math::         
+        \varphi(t; \alpha, p_1, c_P, \mu_P) = e^{ c_P(it\mu_P -|t|^{\alpha}e^{ -i\alpha (p_1 - p_2) \operatorname{sign}(t)\pi/2}}
+
+    where:
+              
+    ..math::
+        p_1 + p_2 = 1 and 
+        0 <= p_1, p_2 <= 1 for 0 < \alpha < 1, 
+        1 / \alpha <= p_1, p_2 <= 1 - 1 / \alpha for 1 < \alpha < 2. 
+
+        p_1 = \frac{1 + \delta}{2} = \frac{1}{2} + \frac{\beta_B K(alpha)}{2\alpha} = 
+            \frac{1}{2} + \frac{1}{\alpha \pi} atan(\beta_A \tan[\pi\alpha / 2])
+         c_P = c_C = c_B = \frac{c_A}{\cos[\beta_B K(\alpha)\pi /2}
+
+    The parameterization P has rather intuitive relation to CDF of stable law. 
+    The parameter p_1 = 1 - CDF(0, \alpha, p_1, c), i.e. the parameter p_1 gives value 
```

Comment:
Underscores have special meaning in reST (references), thus they need to be shielded: either wrap the expression in double backticks (typesets as code), or a `:math` environment.